features/continuous-integration.md

```diff
@@ -1,2 +1,135 @@
+vulnerabilities and Kubernetes misconfigurations early in the development process. Insights
+can scan changes the changes in each pull request, notifying developers or breaking the build
+whenever security, efficiency, or reliability issues are found.
+
+# Setup
+To get started, visit your organization's **Repositories** tab, and click **Add Repo**
+
+<img :src="$withBase('/img/add-repo.png')" alt="action items">
+
+Following the instructions there, add a `fairwinds-insights.yaml` configuration to the base
+of your repository. Here's a minimal example:
+
+> Be sure to replace `acme-co` with your organization's name in Insights.
+```yaml
+options:
+  setExitCode: true
+  organization: acme-co
+
+images:
+  docker:
+  - nginx:1.18-alpine
+  - quay.io/acme-co/my-app:$CI_HASH
+
+manifests:
+  yaml:
+  - ./deploy/test.yaml
+  helm:
+  - name: prod
+    path: ./deploy/chart
+    values:
+      foo: bar
+      resources:
+        requests:
+          memory: 1Gi
+```
+
+Next, you'll need to add the `FAIRWINDS_TOKEN` environment variable to your CI environment. This
+token can be found in the CI instructions in Insights, or on your organization's settings page. Note that
+most CI platforms provide a way to specify secrets in your environment variables.
+
+Last, you'll need to execute the Insights CI script within your CI pipeline.
+You may want to download, inspect, and store a copy of the script in your repository.
+```
+curl https://insights.fairwinds.com/static/insights-ci.sh | bash
+```
+
+Your repository will show up in the Insights UI once that script has been successfully run.
+
+## fairwinds-insights.yaml
+Here's a full list of options available in fairwinds-insights.yaml:
+
+### Options
+These are high-level options for the Insights CI integration.
+* `options.organization` - the name of your organization in Insights
+* `options.setExitCode` - set a non-zero exit code if issues are found. Default `false`
+* `options.baseBranch` - the branch to compare results to. Default `master`
+* `options.newActionItemThreshold` - the maximum number of new action items that can appear for a passing test. Default `0`
+* `options.severityThreshold` - if _any_ action item hits this threshold, the test will fail. Default `danger`
+* `options.junitOutput` - if specified, Insights will print JUnit XML results here
+* `options.repositoryName` - the name of the repository. Must match the name in GitHub if using the GitHub integration. Defaults to the output of `git remote -v`
+* `options.tempFolder` - a temporary directory to store files in. Default `./_insightsTemp/`
+* `options.hostname` - the host of the Insights instance to send results to. Default `https://insights.fairwinds.com`
+
+### Images
+Specify any images you'd like Insights to scan for vulnerabilities. These images must be available
+locally in your CI environment, either through running `docker build` or `docker pull`.
+
+* `images.docker` - an array of image names and tags to scan
```

Comment:
The images section I think there's still a folder option, so they can export their own images out of podman, cri-o, etc and we can still use them.